src/core/permissions/sagas/updatePermissionsBulk.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * @flow
+ */
+
+import {
+  all,
+  call,
+  put,
+  takeEvery,
+} from '@redux-saga/core/effects';
+import { List } from 'immutable';
+import { AxiosUtils, LangUtils, Logger } from 'lattice-utils';
+import type { Saga } from '@redux-saga/core';
+import type {
+  Ace,
+  ActionType,
+  UUID,
+} from 'lattice';
+import type { WorkerResponse } from 'lattice-sagas';
+import type { SequenceAction } from 'redux-reqseq';
+
+import { updatePermissionsWorker } from './updatePermissions';
+
+import { UPDATE_PERMISSIONS_BULK, updatePermissions, updatePermissionsBulk } from '../actions';
+
+const { toSagaError } = AxiosUtils;
+const { isDefined } = LangUtils;
+
+const LOG = new Logger('PermissionsSagas');
+
+function* updatePermissionsBulkWorker(action :SequenceAction) :Saga<WorkerResponse> {
```

Comment:
Instead of `updatePermissionsBulk` saga + action + reducer, how about update `updatePermissions` to take an array?